README.md

```diff
@@ -893,24 +893,24 @@ Table of Contents
 
 ## PaaS
 
-  * [anvil.works](https://anvil.works) - Web app development with nothing but Python. Free tier with unlimited apps.
-  * [appharbor.com](https://appharbor.com/) — A .Net PaaS that provides 1 free worker
+  * [anvil.works](https://anvil.works/) - Web app development with nothing but Python. Free tier with unlimited apps.
+  * [appharbor](https://appharbor.com/) — A .Net PaaS that provides 1 free worker
```

Comment:
lots of good changes here but I'd rather not remove the TLD like here - especially as there's a mix and match only some being added or removed, so would rather just leave as it and keep to fixing the content

thanks!

Reply:
OK I'll delete those changes